SafeguardDevOpsService/SafeguardDevOpsService.csproj

```diff
@@ -76,6 +76,7 @@
     <PackageReference Include="Swashbuckle.AspNetCore.Newtonsoft" Version="5.5.1" />
     <PackageReference Include="Topshelf" Version="4.2.1" />
     <PackageReference Include="Topshelf.Serilog" Version="4.2.1" />
+    <PackageReference Include="WiX" Version="3.11.2" />
```

Comment:
You shouldn't need this either.  It can be reverted.

Reply:
again, VS2019 FTW!